src/Chunk.cpp

```diff
@@ -1643,21 +1643,17 @@ void cChunk::SetAreaBiome(int a_MinRelX, int a_MaxRelX, int a_MinRelZ, int a_Max
 
 void cChunk::CollectPickupsByPlayer(cPlayer & a_Player)
 {
-	double PosX = a_Player.GetPosX();
-	double PosY = a_Player.GetPosY();
-	double PosZ = a_Player.GetPosZ();
+	auto BoundingBox = cBoundingBox(a_Player.GetPosition(), a_Player.GetWidth(), a_Player.GetHeight());
```

Comment:
Side note: there should totally be a `cEntity::GetBoundingBox`.

Reply:
I thought of that, but there's only one other place in the whole codebase where it would be used. Still worth it?

Reply:
I'd say yes. It also looks like it should actually be `GetWidth()/2.0`:
https://github.com/cuberite/cuberite/blob/0b9b7bc1a8d5cf6f92b802dc376a189ef066e62d/src/Entities/Minecart.cpp#L57

Reply:
Unless that was intentionally twice the entity radius?

Reply:
No, it should just have been the entity radius. I guess with only 0.4 blocks difference it behaves basically the same anyway. I'll make a new PR to fix that and create a new method to get the bounding box.